packages/itwin/tree-widget/src/components/trees/imodel-content-tree/IModelContentTree.tsx

```diff
@@ -63,7 +63,7 @@ export const IModelContentTree = (props: IModelContentTreeProps) => {
         state={state}
         selectionMode={selectionMode ?? SelectionMode.None}
         treeRenderer={(treeProps) =>
-          hierarchyLevelFilteringProps ? (
+          hierarchyLevelConfig ? (
```

Comment:
I feel like it would be more correct to check `hierarchyLevelConfig?.isFilteringEnabled` here